components/board/pi/impl/board_test.go

```diff
@@ -240,27 +240,35 @@ func TestPiPigpio(t *testing.T) {
 
 func TestServoFunctions(t *testing.T) {
 	t.Run("check servo math", func(t *testing.T) {
-		pw := angleToPulseWidth(1)
+		pw := angleToPulseWidth(1, 180)
```

Comment:
[nit] better to have the defaults as constant at the top of the file, here and in `components/board/pi/impl/servo.go`